go/ct/spc/specification.go

```diff
@@ -1652,10 +1672,36 @@ func getAllRules() []Rule {
 		},
 	})...)
 
-	// --- CALL, STATICCALL and DELEGATECALL ---
+	// --- CALL, CALLCODE, STATICCALL and DELEGATECALL ---
 
 	rules = append(rules, getRulesForAllCallTypes()...)
 
+	rules = append(rules, Rule{
```

Comment:
This rule defines a test case and is not a specification rule. It should not be part of the specification but of a set of test examples. Rule of thumb: if you fix values like `Eq(Gas(), 1234)` it is likely not a general rule but a test case.

Currently, we have a few condition-derived tests in [`evm_test.go`](https://github.com/Fantom-foundation/Tosca/blob/7f15c9ec604d53f7448289dd41767c86650db4d5/go/ct/evm_test.go#L37). We could extend them or even add a rule-derived section. This would get those tests covered.

However, this will not solve the problem that these type of cases are not tested by CT runs. The only solution for this is to refine the specification conditions by partitioning the state space using Cartesian products of conditions.

Reply:
discussed offline, we concluded it is best to remove this rule.